backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/BuiltInFictitiousFunctionIrClassFactory.kt

```diff
@@ -48,6 +48,16 @@ internal class BuiltInFictitiousFunctionIrClassFactory(
 
     class FunctionalInterface(val irClass: IrClass, val arity: Int)
 
+    fun buildAllClasses() {
+        val maxArity = 255 // See [BuiltInFictitiousFunctionClassFactory].
```

Comment:
Where is BuiltInFictitiousFunctionClassFactory?

Reply:
https://github.com/JetBrains/kotlin/blob/c972bf5882977c8b1ae2684602f3686dd2069f9b/core/descriptors/src/org/jetbrains/kotlin/builtins/functions/BuiltInFictitiousFunctionClassFactory.kt#L48

See also: https://github.com/JetBrains/kotlin/blob/e5a1040dbd3c39e51eb4713cc7485c9bbff947d0/compiler/ir/serialization.common/src/org/jetbrains/kotlin/backend/common/serialization/KotlinMangler.kt#L305